packages/server-wallet/test/e2e.test.ts

```diff
@@ -0,0 +1,24 @@
+import {spawn, ChildProcessWithoutNullStreams} from 'child_process';
+
+function startServerWallet(port: string): ChildProcessWithoutNullStreams {
+  const server = spawn('yarn', ['ts-node', 'src/index'], {
+    env: {...process.env, NODE_ENV: 'test', PORT: port},
+  });
+
+  server.stdout.on('data', data => console.log(data.toString()));
+  server.stderr.on('data', data => console.error({error: data.toString()}, `Server threw error`));
+
+  return server;
+}
+
+const aliceHttp = startServerWallet('5001');
```

Comment:
Minor point: I don't know that we want Alice to start an HTTP server.

We definitely want to build a virtual funding protocol that works given the constraint that
- Alice can spontaneously message Bob
- Bob can only respond to messages from Alice
where this constraint comes from the fact that Bob is serving HTTP requests, and Alice is making HTTP requests.

Reply:
Okay; so it would be overkill for this test, and messaged can be provided to Alice directly by calling Wallet.pushMessage then